### PR TITLE
Feature/bee 613 solana nft adapter

### PR DESCRIPTION
**Pull request by ibrica**

### CODE REVIEW

File: packages/solana-nft-adapter/.gitignore [hunk 1]
```diff
@@ -0,0 +1,106 @@
+node_modules/
```
Comment (martinavagyan) [inline packages/solana-nft-adapter/.gitignore:1]: having two gitignores in the repo can be confusing, do you mind merging the two together on the root?

File: packages/solana-nft-adapter/.gitignore [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+!tests/cases/projects/projectOption/**/node_modules
+!tests/cases/projects/NodeModulesSearch/**/*
+!tests/baselines/reference/project/nodeModules*/**/*
+.idea
+yarn.lock
+yarn-error.log
+.parallelperf.*
+tests/cases/user/*/package-lock.json
+tests/cases/user/*/node_modules/
+tests/cases/user/*/**/*.js
```
Comment (martinavagyan) [inline packages/solana-nft-adapter/.gitignore:82]: I think this file is generated somehow, you can cover most of those ignore cases with `scripts/` etc

File: packages/solana-nft-adapter/test/solanaNft.ts [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    console.log('Here');
+
+    buyerCurrencyAccount = await createAssociatedTokenAccount(
+      provider.connection,
+      payer,
+      currencyKey,
+      buyerMainAccount.publicKey
+    );
+
+    console.log('Buyer currency account: ', buyerCurrencyAccount.toString());
```
Comment (martinavagyan) [inline packages/solana-nft-adapter/test/solanaNft.ts:168]: console logs

File: packages/solana-nft-adapter/.gitignore (hunk 1 above)
Comment (ibrica) [reply packages/solana-nft-adapter/.gitignore]: removed

File: packages/solana-nft-adapter/.gitignore (hunk 2 above)
Comment (ibrica) [reply packages/solana-nft-adapter/.gitignore]: yes, removed second .gitignore

File: packages/solana-nft-adapter/test/solanaNft.ts (hunk 3 above)
Comment (ibrica) [reply packages/solana-nft-adapter/test/solanaNft.ts]: removed, except one that I need to see balance info when tests running

**Review by martinavagyan — APPROVED** [review]

**Pull request merged**